pyart/aux_io/rainbow_wrl.py

```diff
@@ -0,0 +1,493 @@
+"""
+pyart.aux_io.rainbow
+====================
+
+Routines for reading RAINBOW files (Used by SELEX) using the wradlib library
+
+This routine has been tested to read rainbow5 files version 5.22.3, 5.34.16 and 5.35.1.
+Since the rainbow file format is evolving constanly there is no guaranty that it can work with other versions.
+If necessary, the user should adapt to code according to its own version.
+
+Data types read by this routine:
+Reflectivity: dBZ, dBuZ, dBZv, dBuZv
+Velocity: V, Vu, Vv, Vvu
+Spectrum width: W, Wu, Wv, Wvu
+Differential reflectivity: ZDR, ZDRu
+Co-polar correlation coefficient: RhoHV, RhoHVu
+Co-polar differential phase: PhiDP, uPhiDP, uPhiDPu
+Specific differential phase: KDP, uKDP, uKDPu
+Signal quality parameters: SQI, SQIu, SQIv, SQIvu
+Temperature: TEMP
+Position of the range bin respect to the ISO0: ISO0
+
+History
+
+    V0.1 20160629 Jordi Figueras i Ventura, MeteoSwiss, first prototype
+        
+    read_rainbow_wrl
+
+"""
+
+# specific modules for this function
+import os
+import wradlib as wrl
```

Comment:
wradlib is an optional dependency in Py-ART and therefore it cannot be imported unconditionally.  You can add a try/except block similar to the one used in the `sinarame_h5.py` module for importing h5py:

``` Python
try:
    import wradlib as wrl
    _WRADLIB_AVAILABLE = True
except:
    _WRADLIB_AVAILABLE = False
```